### PR TITLE
Darken Overlay Background for Light Video Backgrounds

### PR DESCRIPTION
**Pull request by esegebart**
Changed the opacity to be twice as dark so if there is a very light background the controls do not get lost against the video. 
The video is still able to be seen underneath the overlay.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,22 +24,26 @@
     "prepare": "bob build",
     "release": "release-it"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@babel/core": "^7.20.12",
+    "@babel/preset-env": "^7.1.6",
+    "@types/react-native": "^0.63.17"
+  },
```
Comment (criszz77) [inline package.json:31]: Unfortunately, these should be removed. Is this something that you needed?

**Review by criszz77 — CHANGES_REQUESTED** [review]

**Comment by criszz77** [comment]
Looks great! I'll test it tomorrow on my project and see how it looks. Thank you!

**Comment by esegebart** [comment]
Oh ! Nope I do not need those. I think I added them when I was trying to load the project thinking I needed them, but I don't! I will remove !

**Comment by criszz77** [comment]
Don’t worry, just asking. Unfortunately, I didn’t have time to test changes, but I will definitely do it tomorrow. Have a nice weekend!

**Comment by esegebart** [comment]
@criszz77  If you have time, could you tell me if there is any way to debug / console.log when running or using the media console project? I am trying to understand where data is passed through to see how everything works. Otherwise you have a nice weekend also and I will touch base with you on Monday!

**Comment by criszz77** [comment]
Sure. The normal way should work. Just do your changes, and then follow the instrunctions. The logs should appear where you normally check for logs.

**Comment by esegebart** [comment]
@criszz77 Awesome, thank you so much for your help. :)

**Comment by criszz77** [comment]
Hey

Sorry for not giving too much attention as promised... but this week we'll definitely merge it

**Pull request closed without merging**